.github/workflows/core-test.yml

```diff
@@ -0,0 +1,45 @@
+name: neo.core
+
+on:
+  pull_request:
+    branches: [master]
+    types: [synchronize, opened, reopened, ready_for_review]
+
+jobs:
+  multi-os-python-numpy:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: true
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+        python-version: ['3.7', '3.8', '3.9']
+        numpy-version: ['1.16.6', '1.19.5', '1.20.3', '1.21.5', '1.22.3']
+        exclude:
```

Comment:
What is the issue with this combination? Maybe put a comment here, so we know when to remove this constraint....

Reply:
It is not an issue but the fact that we run 45 time sthe same test!!
Some numpy are not available for some python.